docs/support_table.md

```diff
@@ -801,7 +801,7 @@ use `\ce` instead|
 |\ominus|$\ominus$||
 |\operatorname|$\operatorname{asin} x$|`\operatorname{asin} x`|
 |\operatorname\*|$\operatorname*{asin}\limits_y x$|`\operatorname*{asin}\limits_y x`|
-|\operatornamewithlimits$\operatornamewithlimits{asin}\limits_y x$|`\operatornamewithlimits{asin}\limits_y x`|
+|\operatornamewithlimits$\operatornamewithlimits{asin}\limits_y x$|`\operatornamewithlimits{asin}\limits_y x`||
```

Comment:
This still needs work. It now displays with the function name and the rendered version in the same cell. What you need is:

```
|\operatornamewithlimits | $\operatornamewithlimits{asin}\limits_y x$|`\operatornamewithlimits{asin}\limits_y x`|
```

Reply:
Oops, sorry, I was clearly looking at this too quickly!